nexmo/__init__.py

```diff
@@ -181,3 +259,31 @@ def parse(self, host, response):
       message = "{code} response from {host}".format(code=response.status_code, host=host)
 
       raise ServerError(message)
+
+  def __get(self, request_uri, params={}):
+    uri = 'https://' + self.api_host + request_uri
+
+    return self.parse(self.api_host, requests.get(uri, params=params, headers=self.__headers()))
+
+  def __post(self, request_uri, params):
+    uri = 'https://' + self.api_host + request_uri
+
+    return self.parse(self.api_host, requests.post(uri, json=params, headers=self.__headers()))
```

Comment:
Looks like JWT headers are always sent even if the API doesn't support JWT.

@tjlytle I'm guessing they'll just be ignored. So no big deal.

Same for `__get` and `__put`